api/src/main/java/jakarta/ejb/ActivationConfigProperty.java

```diff
@@ -35,7 +35,7 @@
  * which messages a JMS message driven bean is to receive.
  *
  * <li><code>destinationType</code>. This property is used to specify whether the message driven bean is intended to be
- * used with a queue or a topic. The value must be either <code>javax.jms.Queue</code> or <code>javax.jms.Topic</code>.
+ * used with a queue or a topic. The value must be either <code>jakarta.jms.Queue</code> or <code>jakartaF.jms.Topic</code>.
```

Comment:
```suggestion
 * used with a queue or a topic. The value must be either <code>jakarta.jms.Queue</code> or <code>jakarta.jms.Topic</code>.
```

Reply:
Thanks for catching it up.
Big fingers lol